kurl_util/cmd/pvmigrate/main.go

```diff
@@ -52,17 +64,51 @@ func main() {
 		os.Exit(1)
 	}
 
-	if !skipFreeSpaceCheck {
+	if !skipFreeSpaceCheck || dryRun {
 		if err := checkFreeSpace(ctx, logger, cfg, cli, opts); err != nil {
-			logger.Printf("Failed to check cluster free space: %s", err)
+			logger.Printf("failed to check cluster free space: %s", err)
 			os.Exit(1)
 		}
 	}
 
-	if err = migrate.Migrate(ctx, logger, cli, opts); err != nil {
-		fmt.Printf("%s\n", err.Error())
-		os.Exit(1)
+	if !skipPvAccessModeCheck || dryRun {
+		unsupportedPVCs, err := validatePVAccessMode(ctx, logger, cfg, cli, opts)
+		if err != nil {
+			logger.Printf("failed to validate access modes for destination storage provider %s: %s", opts.DestSCName, err)
+			os.Exit(1)
```

Comment:
same (return the error ideally or use logger.Fatalf() if required`: https://github.com/replicatedhq/kURL/pull/3767/files#r1030680619

Reply:
I've update this.